code/mob/living/critter/icephoenix.dm

```diff
@@ -0,0 +1,322 @@
+		else
+			boutput(src, SPAN_NOTICE("You will no longer travel back to station space when traveling off the Z level"))
+
+	proc/radiate_cold(turf/center)
+		var/obj/phoenix_snow_floor/floor
+		var/icon_s
+		var/direct
+		for (var/turf/space/T in block(center.x - 1, center.y - 1, center.z, center.x + 1, center.y + 1, center.z))
+			floor = locate() in T
+
+			if (floor)
+				icon_s = floor.icon_state
+				direct = floor.dir
+				qdel(floor)
+			floor = new /obj/phoenix_snow_floor(T)
+			if (icon_s)
+				floor.icon_state = icon_s
+				floor.set_dir(direct)
+
+/obj/phoenix_snow_floor
+	name = "compacted snow floor"
+	desc = "A floating layer of compacted snow and ice in space. How is that even possible?"
+	icon = 'icons/turf/snow.dmi'
+	icon_state = "snow1"
+	layer = PLATING_LAYER
+	plane = PLANE_UNDERFLOOR
+	anchored = ANCHORED_ALWAYS
+	stops_space_move = TRUE
+	alpha = 160
+
+	New()
+		..()
+		src.icon_state = pick("snow1", "snow2", "snow_rough1")
+		src.set_dir(pick(cardinal))
+		SPAWN(6 SECONDS)
+			animate(src, 6 SECONDS, alpha = 80)
+		SPAWN(12 SECONDS) // 45 SECONDS
+			qdel(src)
+
+/turf/simulated/ice_phoenix_ice_tunnel
+	icon = 'icons/mob/critter/nonhuman/icephoenix.dmi'
+	icon_state = "ice_tunnel"
+	density = FALSE
+	opacity = FALSE
+	name = "ice tunnel"
+	desc = "A narrow ice tunnel that seems to prevent passage of air by a thick, icy mist. Interesting."
+	gas_impermeable = TRUE
+
+	New(newLoc, direct)
+		src.dir = direct
+		if (src.dir == NORTH)
+			src.dir = SOUTH
+
+		if (src.dir == NORTH || src.dir == SOUTH)
+			src.blocked_dirs = EAST | WEST
+		else
+			src.blocked_dirs = NORTH | SOUTH
+		..()
+
+/obj/ice_phoenix_statue
+	icon = 'icons/mob/critter/nonhuman/icephoenix.dmi'
+	icon_state = "icephoenix"
+	name = "ice statue"
+	desc = "Some sort of ice statue resembling a phoenix. It's emanating an aura."
+	density = TRUE
+	anchored = ANCHORED_ALWAYS
+	color = "#0400da"
+	var/health = 200
+
+	attack_hand(mob/user)
+		attack_particle(user, src)
+		user.lastattacked = src
+		boutput(user, SPAN_NOTICE("It's really cold!"))
+		if (!ON_COOLDOWN(src, "hit_impact_sound", 2 SECONDS))
+			playsound(get_turf(src), 'Glass_Shards_Hit_1.ogg', 75, TRUE)
+		..()
+
+	attackby(obj/item/I, mob/user)
+		..()
+		attack_particle(user, src)
+		user.lastattacked = src
+
+		if (!I.force)
+			return
+
+		hit_twitch(src)
+		src.health -= I.force
+		if (src.health <= 0)
+			qdel(src)
+			return
+
+		if (!ON_COOLDOWN(src, "hit_impact_sound", 2 SECONDS))
+			playsound(get_turf(src), 'Glass_Shards_Hit_1.ogg', 75, TRUE)
+
+	bullet_act(obj/projectile/P)
+		if (istype(P.proj_data, /datum/projectile/bullet/ice_phoenix_icicle))
+			return
+		if (P.proj_data.ks_ratio >= 1)
+			hit_twitch(src)
+			src.health -= P.power
+			if (src.health <= 0)
+				qdel(src)
+				return
+			if (!ON_COOLDOWN(src, "hit_impact_sound", 2 SECONDS))
+				playsound(get_turf(src), 'Glass_Shards_Hit_1.ogg', 75, TRUE)
```

Comment:
```suggestion
				playsound(get_turf(src), 'sound/impact_sounds/Glass_Shards_Hit_1.ogg', 75, TRUE)
```
![test your code](https://github.com/user-attachments/assets/647ac546-26bd-4713-92ac-5baf9f349781)
(same for the rest of these in this file)

Reply:
oops